skin.json

```diff
@@ -147,11 +148,11 @@
   },
   "controlBar": {
     "volumeControl": {
-      "color": "rgba(67,137,255,1)"
+      "color": ""
     },
     "iconStyle": {
       "active": {
-        "color": "#FFFFFF",
```

Comment:
@mistermendez As discussed with you over the chat, this was the requirement from Steve and David